packages/mdx/mdx-hast-to-jsx.js

```diff
@@ -112,7 +121,11 @@ function toJSX(node, parentNode = {}, options = {}) {
 
     return <MDXTag
              name="wrapper"
-             ${layout ? `Layout={this.layout} layoutProps={props}` : ''}
+             ${
+               layout
+                 ? `Layout={this.layout} layoutProps={{ ...layoutProps, ...props }}`
```

Comment:
I think we might need to tweak this to use `Object.assign` or configure `buble` to polyfill:

```suggestion
                 ? `Layout={this.layout} layoutProps={Object.assign(layoutProps, props)}`
```